test/opensearch/test_opensearch.py

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (C) 2025-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+#
+from opensearchpy import OpenSearch
+import pytest
+
+@pytest.mark.asyncio
+async def test_opensearch_basic():
+    host = 'localhost'
+    port = 9200
+
+    # Create the client with SSL/TLS and hostname verification disabled.
+    client = OpenSearch(
```

Comment:
This should be done is some `OpenSearchManager`, which should be passed to a test as an argument. From the manager, the test should be able to get client to the OpenSearch, without specifying host, port and other parameters manually.

Reply:
Marking as resolved as following commit fixes this.